getter.go

```diff
@@ -144,6 +144,7 @@ func (g *getter) queueFile(url *url.URL) (http.Header, error) {
 	// resp could be nil, depending on the error.
 	if err != nil {
 		g.chunkWg.Done()
+		defer resp.Body.Close()
```

Comment:
Thats where i originally looked however the response here is re-used in init chunks (remember back to when we made this re-use the first request for the first chunk)

I think ideally ```g.retryRequest()``` would copy the body to a new ```[]byte``` and close the response itself, however i didnt want to refactor so many lines in something we're pushing out so quick. Let me know what you think